fla/ops/oja2/chunk_h.py

```
@@ -0,0 +1,821 @@
+    u += ((bos * H + i_h) * K).to(tl.int64)
+    w += ((bos * H + i_h) * V).to(tl.int64)
+    if SAVE_NEW_KEY:
+        k_new += ((bos * H + i_h) * K).to(tl.int64)
+    stride_v = H*V
+    stride_h = H*K*V
+    stride_k = H*K
+    if USE_INITIAL_STATE:
+        h0 = h0 + i_nh * K*V
+    if STORE_FINAL_STATE:
+        ht = ht + i_nh * K*V
+    BV=64
```

Flake8 (7.3.0)

[error] local variable 'BV' is assigned to but never used

(F841)